macos/FluentUITestViewControllers/TestButtonViewController.swift

```diff
@@ -6,134 +6,340 @@
 			]
 		}
-		
-		let customDisabledButton = Button()
-		customDisabledButton.title = "DisabledWithImg"
-		customDisabledButton.contentTintColor = .white
-		customDisabledButton.restBackgroundColor = Colors.Palette.blueMagenta30.color
-		customDisabledButton.image = leadingArrowImage
-		customDisabledButton.imagePosition = .imageLeading
-		customDisabledButton.isEnabled = false
-		
-		let disabledPrimaryFilled = Button(title: "PrimaryFilled disbaled", style: .primaryFilled)
-		let disabledPrimaryOutline = Button(title: "PrimaryOutline disabled", style: .primaryOutline)
-		let disabledBorderless = Button(title: "Borderless disabled", style: .borderless)
-		disabledPrimaryFilled.isEnabled = false
-		disabledPrimaryOutline.isEnabled = false
-		disabledBorderless.isEnabled = false
-		
-		let disabledButtons: () -> [NSButton] = {
+
+		let disabledButtonsWithTitleAndImageHorizontal = buttonsWithTitleAndImageHorizontal().map { button -> TestButton in
+			button.isEnabled = false
+			return button
+		}
+
+		let buttonsWithTitleAndImageVertical: () -> [TestButton] = {
 			return [
-				disabledPrimaryFilled,
-				disabledPrimaryOutline,
-				disabledBorderless,
-				customDisabledButton,
+				TestButton(title: "Nope", image: stopImage, imagePosition: .imageAbove, format: largePrimary),
+				TestButton(title: "Nope", image: stopImage, imagePosition: .imageBelow, format: largeSecondary),
+				TestButton(title: "Nope", image: stopImage, imagePosition: .imageOverlaps, format: largeAcrylic),
+				TestButton(title: "Nope", image: stopImage, imagePosition: .imageAbove, format: largeBorderless),
+				TestButton(title: "Nope", image: stopImage, imagePosition: .imageBelow, format: largeCustom),
+				TestButton(title: "Nope", image: stopImage, imagePosition: .imageOverlaps, format: smallPrimary),
+				TestButton(title: "Nope", image: stopImage, imagePosition: .imageAbove, format: smallSecondary),
+				TestButton(title: "Nope", image: stopImage, imagePosition: .imageBelow, format: smallAcrylic),
+				TestButton(title: "Nope", image: stopImage, imagePosition: .imageOverlaps, format: smallBorderless),
+				TestButton(title: "Nope", image: stopImage, imagePosition: .imageAbove, format: smallCustom),
 			]
 		}
 
-		let gridView = NSGridView(frame: .zero)
-		gridView.rowSpacing = TestButtonViewController.gridViewRowSpacing
-		gridView.columnSpacing = TestButtonViewController.gridViewColumnSpacing
-		gridView.setContentHuggingPriority(.defaultHigh, for: .vertical)
-		
-		gridView.addColumn(with: columnLabels)
-		gridView.addColumn(with: buttonsWithTitle())
-		gridView.addColumn(with: buttonsWithImage())
-		gridView.addColumn(with: buttonsWithTitleAndImage())
-		gridView.addColumn(with: disabledButtons())
-
-		let emptyCell = NSGridCell.emptyContentView
-	
-		// Insert the Row Titles as the last step
-		let rowLabels: [NSView] = [
-			emptyCell,
+		let fluentButtonsGrid = NSGridView(frame: .zero)
+		fluentButtonsGrid.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
ha, of course I forgot to set this before. That might explain some of the weirdness with the grid view before